src/actions/index.js

```diff
@@ -84,7 +86,25 @@ export const getSchoolsFailure = () => ({
   type: actionTypes.GET_SCHOOLS_FAILURE,
 });
 
+//////
```

Comment:
Try not to leave these in when you put up PRs (having stuff to help you debug/develop locally is fine)

Reply:
Will keep in mind for the future.